docs_src/examples/Ch-OSImageWithEdgeX.md

```diff
@@ -905,8 +905,8 @@ If you haven't done so, follow the instructions [here](https://snapcraft.io/docs
 Read [here](https://ubuntu.com/core/docs/system-user) to know how the manual account setup looks like and how it can be automated.
 
 ## References
-- [Getting Started using Snaps](https://docs.edgexfoundry.org/2.2/getting-started/Ch-GettingStartedSnapUsers)
-- [EdgeX Core Data](https://docs.edgexfoundry.org/2.2/microservices/core/data/Ch-CoreData/)
+- [Getting Started using Snaps](https://docs.edgexfoundry.org/{{version}}/getting-started/Ch-GettingStartedSnapUsers)
+- [EdgeX Core Data](https://docs.edgexfoundry.org/{{version}}/microservices/core/data/Ch-CoreData/)
```

Comment:
These should a relative links since they are part of the docs.
```suggestion
- [Getting Started using Snaps]../../(getting-started/Ch-GettingStartedSnapUsers/)
- [EdgeX Core Data](../../microservices/core/data/Ch-CoreData/)
```